src/pages/devtools/componnents/FormsValuesReader.tsx

```diff
@@ -0,0 +1,58 @@
+import { useState, useEffect } from 'react';
+import CurrentUrl from './CurrentUrlComponnent';
+
+const BASE_URL = CurrentUrl + '/Form';
```

Comment:
This code is even working? You get currentUrl but from implementation, it seems that it is not returning it. As well you use the BASE_URL in line `23` but you add the `/Form` twice.